ci/gpu/build.sh

```diff
@@ -40,6 +40,10 @@ conda list
 # BUILD - Build libcudf and cuDF from source
 ################################################################################
 
+# Temporarily install feather for testing
+logger "conda install -c rapidsai-nightly nvstrings=0.3.0dev0"
+conda install -c rapidsai -c rapidsai-nightly -c conda-forge -y openssl=1.0.2r nvstrings=0.3.0dev0
```

Comment:
rapidsai-nightly channel will by default pull cuda9.2 packages. You'll want to programmatically define which cuda version you're using in the script here. I did something similar in the /ci/gpu/build.sh in cuGraph, so check that out for reference.

Reply:
Yea this was just for testing if I could get only the CUDA 9.2 builds to not have openssl errors but it still didn't work anyway. Think I figured out the conda recipe issue though.